### ISSUE TITLE
jsdom-react: Click on checkbox or radio does not trigger change event

### ISSUE THREAD

**Issue by zonzujiro** [wix-incubator/unidriver#90]
UniDriver will simulate `click` using `Simulate` from `react-dom/test-utils`.
Simulate, will create instance of `SyntheticEvent` and dispatch it, using internal logic of `ReactDOM`. And thats all. No actual click on `input` - no `change` event.

And in jsdom emits click, input, change events in order after dispatching click event.
You can see test case [here](https://github.com/jsdom/jsdom/blob/d6f8a97b5fb7709d6ad0215c1ae95fd4cab58489/test/web-platform-tests/to-upstream/html/semantics/forms/the-input-element/radio-click-events.html#L36) as an example.

Is it possible to simulate behaviour similar to behaviour of browser and trigger `change` event?

**Comment by GabiGrin** [comment]
hmm.. that's why I really wanted to avoid simulate whenever possible
I do think it makes sense to emit this event as well, similar to what we've done with mouse events & click
@zonzujiro  any chance you can help with that? adding a failing test & fix shouldn't be too hard

**Comment by zonzujiro** [comment]
@GabiGrin sure I can. But I am not sure what do you mean. Please, can you clarify?

Add new public method `change`? Something like?

```javascript
change: async eventData => {
  const el = await elem();
  
  Simulate.change(el, eventData)
}
```

Or something else?

**Comment by zonzujiro** [comment]
@GabiGrin sure I can, but I am confused about what you mean. Please, can you clarify?

Right now, as I see, mouse events also using `Simulate`.

**Comment by GabiGrin** [comment]
Sure!
For consistency better add simulate change (only for relevant tags).
I'd love to switch all backto native events but it has it's issues (for example, doesn't work well when the component isn't mounted to body).

**Comment by GabiGrin** [comment]
thanks @zonzujiro! LGTM!
PR build fail looks unrelated

@borisd9  @mosheKerbel @shlomitc any objections to merge this?

**Comment by GabiGrin** [comment]
@zonzujiro  @mosheKerbel  FYI released jsdom `@unidriver/jsdom-react:1.5.0`
